test/test_quantized.py

```diff
@@ -950,5 +950,73 @@ def test_qnnpack_linear(self, output_channels, X):
         # Assert equal
         np.testing.assert_array_almost_equal(Y_q_ref2.dequantize().numpy(), Y_q.dequantize().numpy(), decimal=4)
 
+
+"""Tests the correctness of the tensor comparators."""
+class TestComparatorOps(TestCase):
+    """Tests the element-wise equality ops."""
+    @given(A=hu.tensor(shapes=((3, 4, 5),),
+                       qparams=hu.qparams(dtypes=torch.quint8)),
+           B=hu.tensor(shapes=((5,), (1, 5), (1, 1, 5), (4, 5), (3, 4, 5)),
+                       qparams=hu.qparams(dtypes=torch.quint8)))
```

Comment:
Can you sweep over dtypes to include qint8 and qint32? Does the comparison work for all scalar types (int/float/double?)

Reply:
In the context of the current implementation it doesn't matter, because the implementation involves dequantization, any data type would work. In the upcoming PRs, I will add more efficient implementations, and appropriate randomized tests for them